server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -831,3 +831,9 @@ timestamp_precision=ms
 # time range for partitioning data inside each storage group, the unit is second
 # Datatype: long
 # partition_interval=604800
+
+# admin username, default is root
+admin_name=root
```

Comment:
We'd better keep these comments in a consistent format.
Such as 
```
# admin username, default is root
# Datatype: string
# admin_name=root
```

Reply:
done